nss-day-cohort-48/Rare-API-Spicy-Sauce-Scouts
# Tlc profile view

#### Changes Made
1. Added files for authors directory.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout tlc-profile-view
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Run server
4. See separate client side request

5. View code files for authors, see if anything looks crazy
